docs/designers-developers/developers/data/data-core-editor.md

```diff
@@ -382,6 +382,18 @@ _Related_
 
 -   getFirstMultiSelectedBlockClientId in core/block-editor store.
 
+<a name="getFootnotes" href="#getFootnotes">#</a> **getFootnotes**
```

Comment:
Is this something we could use custom sources for (#16402), rather than to build-in awareness of footnotes to the editor module?

Reply:
I don't have a super deep understanding of custom sources. Would it work if the source is the post content?